evennia/commands/default/building.py

```diff
@@ -126,7 +128,17 @@ def parse(self):
                     aliases = [alias.strip() for alias in aliases.split(";") if alias.strip()]
                 if "/" in objdef:
                     objdef, attrs = [part.strip() for part in objdef.split("/", 1)]
-                    attrs = [part.strip().lower() for part in attrs.split("/") if part.strip()]
+                    _attrs = []
+                    for match in (
+                        match
+                        for part in map(str.strip, attrs.split("/"))
+                        if part and (match := _KEY_REGEX.match(part.strip()))
+                    ):
+                        attr = match.group("attr").lower()
+                        if match.group("key"):
+                            attr += match.group("key")
+                        _attrs += [attr]
```

Comment:
Please use `_attrs.append(attr)` here, it's not only faster than using the add operator, it's also more Pythonic. :)

Reply:
done!